jscomp/outcome_printer/reason_syntax_util.ml

```diff
@@ -0,0 +1,524 @@
+(* Hello! Welcome to the Reason syntax util logic.
```

Comment:
Don't worry about this file too much. Most of it is cppo-ed out, except two helpers at the beginning of the file. I kept it this way for easier syncing.